pkg/internal/token/cacheexportreplace.go

```diff
@@ -0,0 +1,81 @@
+package token
```

Comment:
Not sure i understand the current name of the file `cacheexportreplace.go` can we just name `msalcacheprovider.go` ?